CONTRIBUTING.md

```diff
@@ -140,6 +140,13 @@ than 1, it is clamped to 1 at minimum.
 This is not meant for use by users, but can be suggested in diagnosing an issue
 should one arise with the backtrack limits.
 
+### `RUSTUP_MAX_RETRIES`
+
+When downloading a file, rustup will retry the download a number of times. The
+default is 3 times, but if this variable is set to a valid usize then it is the
+max retry count. A value of `0` means no retries, thus the default of `3` will
+mean a download is tried a total of four times before failing out.
```

Comment:
Hm, shouldn't this go to Readme instead?

Reply:
My hope is that noone will need to tweak this, if we decide to expose it to users then it becomes a thing we have to support long term, rather than something where in an issue we can say "Please try this and see if it helps" to help decide if we need to change the defaults.

Reply:
Ahhh...

I think a valid use-case might be increasing the limit for CI (where networking can get unstable). Cargo has a knob for it in `.cargo/config`, `[net] retries = 92`.

Reply:
CI is a place where people get *even more* angry if I change things.  Humans seem more prepared to change their own workflows than to change CI setups which "have always worked".